vscode-dotty/src/extension.ts

```diff
@@ -77,15 +74,57 @@ function fetchAndRun(artifact: string) {
         throw new Error(msg)
       }
 
-      run({
-        command: "java",
-        args: ["-classpath", classPath, "dotty.tools.languageserver.Main", "-stdio"]
+      configureIDE().then((res) => {
+        run({
+          command: "java",
+          args: ["-classpath", classPath, "dotty.tools.languageserver.Main", "-stdio"]
```

Comment:
Any reason why we extract the classpath with `coursier fetch` and then run this way rather than doing `coursier launch`?

Reply:
We display a progress notification while coursier fetch is running, if we use coursier launch we don't know when downloading has finished.